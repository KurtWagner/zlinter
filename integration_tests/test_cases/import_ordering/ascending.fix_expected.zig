const apple = @import("apple");
const appricot = @import("appricot");
const banana = @import("banana");

// import_order checks per scope, so although this chunk has the wrong order
// it won't be detected until the issues in the above chunk are fixed first.
const cat = @import("cat");
const zebra = @import("zebra");

pub fn main() void {
    const apple_main = @import("apple_main");
    const banana_main = @import("apple_main");

    _ = banana_main;
    _ = apple_main;
}

pub const namespace = struct {
    const apple_namespace = @import("apple_namespace");
    const banana_namespace = @import("apple_namespace");
};
