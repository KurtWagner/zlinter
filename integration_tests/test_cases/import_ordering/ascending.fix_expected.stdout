3 fixes applied to: .zig-cache/tmp/import_ordering.input.zig
Fixed 3 issues in 1 files!
0 issues disabled by comments.
19 fix passes remaining
1 fixes applied to: .zig-cache/tmp/import_ordering.input.zig
Fixed 1 issues in 1 files!
0 issues disabled by comments.
18 fix passes remaining
Fixed 0 issues in 0 files!
0 issues disabled by comments.
Total of 4 issues fixed
